lib/index.js

```diff
@@ -62,7 +65,9 @@ module.exports = function (options) {
 				datadog.increment(`${stat}.response_code.all`, 1, statTags);
 			}
 
-			datadog.histogram(`${stat}.response_time`, new Date() - req._startTime, 1, statTags);
+			if (statTags.length > 0) {
```

Comment:
I would revert this part, this would break the old behavior and I'm fine with submitting the histogram metric even without tags. Any reason for this?

Reply:
I actually added this to respect the existing behavior (and for that matter one of the existing tests).  Currently, if there's no way to determine the route then it bails and doesn't send anything to Datadog.  I figured that not sending the histogram metric if there were no tags to associate the data with was inline with the original author's intent, but if you see value in sending that metric anyway, I'll back it out.